ui/values/strings.go

```diff
@@ -857,4 +858,6 @@ const (
 	StrSetPassword                     = "setPassword"
 	StrPostBond                        = "postBond"
 	StrDCRDEXWelcomeMessage            = "dcrdexWelcomeMessage"
+	StrTradeDex1                       = "tDcrDex1"
+	StrTradeDex2                       = "tDcrDex2"
```

Comment:
I think string variables should have a fragment of what their actual words are. e.g StrExchange = "exchange"